api/app/Actions/EventType/DeleteEventTypeAction.php

```diff
@@ -26,7 +26,7 @@ public function execute(DeleteEventTypeRequest $request): void
             throw new EventTypeNotFoundException('Event Type not found.', 404);
         }
 
-        if ($eventType->owner_id !== Auth::id()) {
+        if ((int)$eventType->owner_id !== Auth::id()) {
```

Comment:
Please remove type casting, the issue with test environment fixed on develop branch.